service/test/lib/integration/teamOpsTest.js

```diff
@@ -153,16 +153,54 @@ lab.experiment('TeamOps', () => {
 
   lab.test('read a specific team', (done) => {
     teamOps.readTeam({ id: '1', organizationId: 'WONKA' }, (err, result) => {
-
       expect(err).to.not.exist()
       expect(result).to.exist()
+      expect(result.usersCount).to.equal(1)
+      expect(result.usersCount).to.equal(result.users.length)
       expect(result.users.length).to.equal(1)
       expect(result.policies.length).to.equal(1)
 
       done()
     })
   })
 
+  lab.test('read users from a specific team', (done) => {
+    teamOps.readTeamUsers({ id: '2' }, (err, result) => {
+      expect(err).to.not.exist()
+      expect(result).to.exist()
+      expect(result.length).to.equal(2)
+      expect(result).to.equal([
+        { id: 'CharlieId', name: 'Charlie Bucket' },
+        { id: 'VerucaId', name: 'Veruca Salt' }
+      ])
+
+      done()
+    })
+  })
+
+  lab.test('paginated read users from a specific team', (done) => {
+    teamOps.readTeamUsers({ id: '2', page: 2, limit: 1 }, (err, result) => {
+      expect(err).to.not.exist()
+      expect(result).to.exist()
+      expect(result.length).to.equal(1)
+      expect(result).to.equal([
+        { id: 'VerucaId', name: 'Veruca Salt' }
+      ])
+
+      done()
+    })
+  })
+
+  lab.test('read users from a specific team, show from the 2nd user', (done) => {
```

Comment:
if you have 2 users and start from the 2nd page where there is 1 user per page you get the 2nd user on the 2nd page. I'll update somehow the text to make it more clear.